src/main/java/pl/betoncraft/betonquest/compatibility/citizens/CitizensInventoryConvIO.java

```diff
@@ -21,13 +24,17 @@ public CitizensInventoryConvIO(final Conversation conv, final String playerID) {
     protected SkullMeta updateSkullMeta(final SkullMeta meta) {
         // this only applied to Citizens NPC conversations
         if (conv instanceof CitizensConversation) {
+            if (Bukkit.isPrimaryThread()) {
+                throw new IllegalStateException("Must be called async!");
```

Comment:
Were all sync calls also logically illegal or why didn't you change anything outside this class?